data/ext/pending/issue-4658-examples.txt

```diff
@@ -0,0 +1,176 @@
+TYPES: #4658 Product, SoftwareApplication
+
+PRE-MARKUP:
+
+<div>
+  <h2>Laptop Battery Replacement</h2>
+  <p>A consumable battery for Model X</p>
+  <p>GTIN: 5901234123457</p>
+  <p>Category: Consumable</p>
+</div>
+
+<div>
+  <h2>Virtual Pump Twin</h2>
+  <p>Digital Twin of the physical water pump for simulation</p>
+  <p>DID: did:web:example.com:product:pump-twin-001</p>
+  <p>Category: Digital Twin</p>
+</div>
+
+<div>
+  <h2>CRM Module</h2>
+  <p>Software module for customer relationship management</p>
+  <p>DID: did:web:example.com:product:crm-module-001</p>
+  <p>Category: Software Component</p>
+</div>
+
+MICRODATA:
+
+<div itemscope itemtype="https://schema.org/Product">
+  <h2 itemprop="name">Laptop Battery Replacement</h2>
+  <p itemprop="description">A consumable battery for Model X</p>
+  <span itemprop="gtin">5901234123457</span>
+  <span itemprop="category">Consumable</span>
+  <div itemprop="additionalProperty" itemscope itemtype="https://schema.org/PropertyValue">
+    <span itemprop="name">Product Type Code</span>
+    <span itemprop="description">UN/CEFACT UNCL 1001 code for Consumable</span>
+    <meta itemprop="propertyID" content="https://service.unece.org/trade/untdid/d19b/tred/tred1001.htm">
```

Comment:
@MatthiasWiesmann sorry, but this has nothing to do here, CEFACT is not an item categorisation standard, it shouldn't be an example here. You absolutely correctly brought alternative examples for something that still, seems basically arbitrarily adding a field to conform with the desired workflow of one company. We had some really good PRs busted here for much less inconsistency. There are absolutely valid fields for this such as category and keywords which can fully model this. If for any reason (that goes absolutely beyond anything I can comprehend) this goes in, at least remove false application of current standards.